### DISCUSSION TITLE
Sonoff basic unable to connect to AP with newer versions.

### DISCUSSION THREAD

**Discussion by atsitras** [arendst/Tasmota#22270] [Q&A]
I am running Tasmota in a bunch of Sonoff basic for a few years now. I have tried to flash from scratch or upgrade some them to version 14.2.0.6, but it fails to connect to AP. Debugging the console i am getting an error that it fails to connect to AP.
Tried multiple SSIDs with different passwords and avoid special characters, without luck in all of the installations.
I am using tasmotizer to flash tasmota.bin.

I have tried multiple versions going down from latest to older versions and i was facing the same connection issue. 
the solution was to go back to version 12.5.0!

Is this a known issue? Is there a workaround?

**Comment by Jason2866** [comment]
Search here in discussions. Everything is already answered
